index.js

```diff
@@ -4,10 +4,12 @@ const auth = require('./routers/auth_router')
 const book = require('./routers/books_router')
 const user_router = require('./routers/user_router')
 const users_router = require('./routers/users_router');
+const categories_router = require('./routers/categories_router')
```

Comment:
Use camelcase for variable naming. Only variables from ./constants we write on snakecase. Recheck all code that all variables in camelcase

Reply:
@SmolinR